src/libecalc/presentation/yaml/yaml_models/pyyaml_yaml_model.py

```diff
@@ -269,8 +273,15 @@ def facility_inputs(self):
         return self._internal_datamodel.get(EcalcYamlKeywords.facility_inputs, [])
 
     @property
-    def time_series(self):
-        return self._internal_datamodel.get(EcalcYamlKeywords.time_series, [])
```

Comment:
what was returned here?  a "string dict"?

Reply:
Previously it was a list of dicts, each dict was a time series entry from yaml